### PR TITLE
landscape.yaml split

### PR DESCRIPTION
**Pull request by Diaphteiros**
`landscape.yaml` is splitted into `landscape_base.yaml` and `user_config.yaml`.

### CODE REVIEW

File: .ci/pipeline_definitions [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ls-setup-template-image:
```
Comment (msohn) [inline .ci/pipeline_definitions]: does this already do anything useful ?

File: components/kubify/terraform.tfvars.aws.template [hunk 2]
```diff
@@ -45,10 +45,7 @@ worker = {
   volume_size = ${config["clusters"]["worker"]["volume_size"]}
```
Comment (msohn) [inline components/kubify/terraform.tfvars.aws.template:45]: I don't understand the purpose of this commit

File: init.sh [hunk 3]
```diff
@@ -36,7 +36,7 @@ export LANDSCAPE_COMPONENTS_HOME="$LANDSCAPE_HOME/setup/components"
 export LANDSCAPE_EXPORT_HOME="$LANDSCAPE_HOME/export"
 
 # merge landscape.yaml if it doesn't exist yet
-if [ ! -f $LANDSCAPE_CONFIG ]; then
+if [ ! -f $LANDSCAPE_CONFIG ] && [ -f $LANDSCAPE_HOME/user_config.yaml ]; then
```
Comment (msohn) [inline init.sh]: how is this commit related to fault tolerance of what ?

File: .ci/pipeline_definitions (hunk 1 above)
Comment (Diaphteiros) [reply .ci/pipeline_definitions]: The idea here is that the ci pipeline is triggered on release of the setup submodule (this project). It will then try to build an image using the dockerfile and push that image into the image repository. It will not actually test/validate the scripts, because this needs a complete environment and will happen when the setup submodule is updated in the landscape-setup-template project.

File: components/kubify/terraform.tfvars.aws.template (hunk 2 above)
Comment (Diaphteiros) [reply components/kubify/terraform.tfvars.aws.template:45]: I was made aware of the fact that while kubify allows to specify AWS credentials for the etcd backup, the landscape-setup-template project doesn't and just uses the AWS credentials written somewhere else in the landscape_config.yaml file (probably the ones from route53). So I changed the landscape_config.yaml file to allow for optional AWS credentials for the etcd backup (this isn't visible in this commit, as the file isn't located in this submodule). However, the credentials being optional means that I can no longer just take "storage_type" and "region" from the etcd configuration, but I'll also have to insert the credentials into the template, if they are given. So I modified both template files to pull all options set in the etcd_backup node from the landscape_config.yaml file, not just storage_type and region. This is what happens in this commit.

**Review by msohn — APPROVED** [review]
Tested successfully, all comments addressed
